server/apps/users/urls.py

```diff
@@ -7,7 +8,10 @@
     path('login/', views.LoginView.as_view()),
     path('user/', views.UserView.as_view()),
     path('logout/', views.LogoutView.as_view()),
-    path('refresh-access-token/', views.RefreshTokenView.as_view()),
+    path('refresh-access-token/', jwt_views.RefreshJSONWebToken.as_view()),
     # to'do app views
-    path('todo-list/', views.TodoListView.as_view()),
+    path('todo/', views.TodoView.as_view()),
+    # to'do functions
+    path('search/', views.SearchTodo.as_view()),
+    path('sort/', views.SortTodo.as_view()),
```

Comment:
Refer https://www.django-rest-framework.org/api-guide/filtering/#api-guide

One API endpoint will work for all usecase

Reply:
Made changes